cmd/authz_mw_cli/authz_mw_cli.go

```diff
@@ -14,14 +14,15 @@ import (
 	"regexp"
 	"strings"
 
-	opamw "github.com/infobloxopen/atlas-authz-middleware/grpc_opa"
-	opacl "github.com/infobloxopen/atlas-authz-middleware/pkg/opa_client"
+	 "github.com/infobloxopen/atlas-authz-middleware/v2/http_opa"
```

Comment:
instead of tying to http_opa, you can introduce a flag in cli to run against http or grpc.

Reply:
added a `TODO` item to add a flag when grpc is supported